.vscode/launch.json

```
@@ -10,8 +10,20 @@
         "--extensionDevelopmentPath=${workspaceFolder}"
       ],
       "outFiles": [
-        "${workspaceFolder}/out"
+        "${workspaceFolder}/out/src"
       ]
+    },
+    {
+      "name": "Tests for Minitest",
```

Just to make this a bit clearer:
```suggestion
      "name": "Run tests for Minitest",
```